File: benchmark/php/fib.php
<?php
function fib($n) {
    if ($n <= 1) return $n;
    return fib($n - 1) + fib($n - 2);
}
$start = hrtime(true);
$result = fib(30);
$end = hrtime(true);
$ms = ($end - $start) / 1e6;
echo "fib(30) = " . $result . "
";
echo "Time: " . round($ms) . "ms
";

File: benchmark/ruby/fib.rb
def fib(n)
  return n if n < 2
  fib(n - 1) + fib(n - 2)
end

start = Time.now.to_f * 1000
result = fib(30)
finish = Time.now.to_f * 1000
puts "fib(30) = #{result}"
puts "Time: #{(finish - start).round}ms"

